src/components/Nav/Nav.js

```diff
@@ -1,7 +1,167 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import styled, { css } from 'styled-components';
 
 function Nav() {
-  return <div />;
+  return (
+    <NavBar>
+      <NavBarContainer>
+        <NavLogo>
+          <Link to="/main">
+            <LogoImg src="/images/logo.svg" alt="워크 폴리오" />
+          </Link>
+        </NavLogo>
+
+        <Menu>
+          <MenuContainer>
+            <Button>
+              <MenuButtonImg
+                src="/images/location.svg"
+                alt="어디로 떠날까요?"
+              />
+              <DestinationText>어디로 떠날까요?</DestinationText>
+            </Button>
+            <Button>
+              <MenuButtonImg src="/images/calendar.svg" alt="언제 떠날까요?" />
+              <CalendarText>언제 떠날까요?</CalendarText>
+            </Button>
+          </MenuContainer>
+
+          <DetailMenu>
+            <Link to="/list-page">
+              <Text> FIND OFFICE </Text>
+            </Link>
+            <Link to="/list-page">
+              <Text> PRE-ORDER</Text>
+            </Link>
+          </DetailMenu>
+
+          <LogInContainer>
+            <Link to="/my-page">
+              <MyPage src="/images/person.svg" alt="마이 페이지" />
+            </Link>
+            <Link to="/login">
+              <LoginText>LOGIN</LoginText>
+            </Link>
+          </LogInContainer>
+        </Menu>
+      </NavBarContainer>
+    </NavBar>
+  );
 }
 
+const flexSpaceBetweenCenter = props => css`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const border = props => css`
+  border-bottom: 1px solid ${props => props.theme.colorLightGray};
+`;
+
+const NavBar = styled.div`
+  padding: 0 60px;
+  ${border}
+  background-color: #fff;
+`;
+
+const NavBarContainer = styled.div`
+  height: 75px;
+  ${flexSpaceBetweenCenter}
+`;
+
+const NavLogo = styled.div`
+  cursor: pointer;
+`;
+
+const LogoImg = styled.img`
+  width: auto;
+  height: 200px;
+  padding-top: 5px;
+`;
+
+const Menu = styled.div`
+  ${flexSpaceBetweenCenter}
+`;
+
+const MenuContainer = styled.div`
+  display: flex;
+  margin-right: 200px;
+`;
+
+const MenuButtonImg = styled.img`
+  display: block;
+  width: 25px;
+  height: 25px;
+  padding-right: 5px;
+  margin-left: 30px;
+  cursor: pointer;
+`;
+
+const DestinationText = styled.p`
+  display: block;
+  padding: 5px;
+  font-size: 16px;
+  font-weight: ${props => props.theme.weightSemiBold};
```

Comment:
```suggestion
  font-weight: ${({theme}) => theme.weightSemiBold};
```
함수 매개변수자리에서 구조분해할당해서 적용할 수 있습니다!

Reply:
오 몰랐던 사실인데 감사합니다 수정하겠습니다!